### ISSUE TITLE
reproducibility addin

### ISSUE THREAD

**Issue by moodymudskipper** [moodymudskipper/myverse#2]
`myverse::myverse_attach()` is not reproducible since it depends on local options, an addin might paste the relevant library and source calls (whether these potential source calls are reproducible is the user's problem from there). 

The package version might be added as a comment after the library call to ease troubleshooting.

**Comment by aghaynes** [comment]
this looks like a nice little package...
talking about reproducibility, it might be better to put the `options(myverse.pkgs = ...)` part in a setup chunk in Rmd or a masterfile (depending on how the project is handled), rather than Rprofile. That might aid portability...
Just a thought... 😃

**Comment by moodymudskipper** [comment]
Good point, it'd be a fair way of using the package. I'll mention it in the README.

**Comment by moodymudskipper** [comment]
Thinking about this more, the addin might not be the best way, the user needs to define it, will forget it exists or which hotkey triggers it etc.

I think an `insert_options` argument to `myverse_attach()` could work, it would use {rstudioapi} to replace current line by an `options(myverse.pkgs=...)` call followed by a `myverse_attach()` call without argument.
